src/openzeppelin/account/account.cairo

```diff
@@ -47,8 +71,9 @@ mod Account {
     use starknet::get_contract_address;
     use zeroable::Zeroable;
 
-    use openzeppelin::account::interface::IDeclarer;
     use openzeppelin::account::interface::ISRC6;
+    use openzeppelin::account::interface::ISRC6Camel;
+    use openzeppelin::account::interface::IDeclarer;
```

Comment:
Any reason to move `IDeclarer` here?

Reply:
To implement the interface in line 129.

Reply:
Haha I meant that it was defined above `interface::ISRC6` and now it's under `ISRC6Camel` 😅 nitpicky question, I know

Reply:
mm, got it, not sure why. Fixed it.